src/components/Recommendation.js

```diff
@@ -190,20 +190,26 @@ function Recommendation() {
         )}
         {projection.options.altitudinalZone &&
           projection.options.targetAltitudinalZone &&
-          projection.options.targetAltitudinalZone.length >= 1 && (
+          projection.options.targetAltitudinalZone.length >= 1 &&
+          projection.options.altitudinalZone !== undefined && (
             <Form.Dropdown
               label={t('targetAltitudinalZone.label')}
               placeholder={t('dropdownPlaceholder')}
               search
               selection
               clearable
               fluid
-              value={targetAltitudinalZone}
+              value={
+                targetAltitudinalZone ||
```

Comment:
What about moving this logic into the lib?

Reply:
we have used the same approach for setTargetAltitudinalZone. If there occurs more of this pattern then it can be moved. (Just my opinion)

Reply:
This smells buggy, but let's move on.